util/stop/stopper.go

```diff
@@ -213,7 +207,7 @@ func (s *Stopper) Stop() {
 }
 
 // ShouldStop returns a channel which will be closed when Stop() has been
-// invoked and outstanding tasks have drained. SetStopped() should be called
+// invoked and outstanding tasks have drained. IsStopped() should be called
```

Comment:
Remove the last sentence; it was trying to convey a "must call" for SetStopped and there's no reason to encourage the use of IsStopped.